etc/portage/package.accept_keywords

```diff
@@ -1,2 +1,5 @@
 media-fonts/dejavu ~x86-macos
 media-fonts/liberation-fonts ~amd64-linux
+sys-cluster/lmod ~amd64
+dev-lua/luaposix ~amd64
+dev-lua/lua-bit32 amd64
```

Comment:
Should this also be `~amd64`? If not, why not?

Can we open an issue to remind ourselves to roll this back when the upstream issue is resolved?

Reply:
This is because the `luaposix` ebuild has `~amd64` (see https://github.com/gentoo/gentoo/blob/master/dev-lua/luaposix/luaposix-35.0-r101.ebuild), while `lua-bit32` has `amd64` (https://github.com/gentoo/gentoo/blob/master/dev-lua/lua-bit32/lua-bit32-5.3.5.1.ebuild). The `~` means "in testing", without it means it's stable. We could also go for `~amd64-linux`, as both have that one, and which is being used for Prefix if I remember correctly.

Reply:
I'll open an issue about undoing this change, then the other one can be closed as soon as this gets merged.

Reply:
See #32.

Reply:
Making it all `~amd64-linux` makes sense to me (as is done above)

Reply:
I was changing this, but then realized that it doesn't work for lmod itself ;-)
https://github.com/gentoo/gentoo/blob/master/sys-cluster/lmod/lmod-8.4.20.ebuild